projects/igniteui-angular/src/lib/select/select.component.spec.ts

```diff
@@ -446,34 +447,42 @@ describe('igxSelect', () => {
     describe('Form tests: ', () => {
         it('Should properly initialize when used as a reactive form control - with validators', fakeAsync(() => {
             const fix = TestBed.createComponent(IgxSelectReactiveFormComponent);
+            let inputGroupWithRequiredAsterisk = fix.debugElement.query(By.css('.' + CSS_CLASS_INPUT_GROUP_REQUIRED));
             fix.detectChanges();
             const selectComp = fix.componentInstance.select;
             const selectFormReference = fix.componentInstance.reactiveForm.controls.optionsSelect;
             expect(selectFormReference).toBeDefined();
             expect(selectComp).toBeDefined();
             expect(selectComp.selectedItem).toBeUndefined();
             expect(selectComp.value).toEqual('');
+            expect(inputGroupWithRequiredAsterisk).toBeDefined();
```

Comment:
That check won't refresh for every expect, since the variable is assigned once and thus the query is ran once.

Reply:
Should probably add a test where the state is not constant (required changes)